tests/test_persist.py

```diff
@@ -16,6 +18,7 @@ def test_pickle(self):
         q = pickle.loads(data)
         self.assertTrue(q.equals(p))
 
+    @shapely20_todo  # big_endian is not yet implemented
```

Comment:
In pygeos right now we didn't implement this keyword, but have a `byte_order` keyword instead (matches more closely the terminology of GEOS). The default is to use the machine's endianness, and one can use the keyword to override this.